docs/bootcamp/openai/index.rst

```diff
@@ -10,4 +10,5 @@ OpenAI
    image_generation
    sql_translate
    image_generation
-   speech_to_text
\ No newline at end of file
+   speech_to_text
+   tweet_classifier
```

Comment:
maybe it's more suitable in the next `sql_translate`, because `image_generation` and `speech_to_text` are multi-model bootcamp examples